l3kernel/l3expan.dtx

```diff
@@ -1345,6 +1346,13 @@
     \exp_after:wN #3
     \exp_after:wN { \exp:w \@@_eval_register:c {#4} }
   }
+\cs_new:Npn \exp_args:NNNe #1#2#3#4
```

Comment:
I wonder, whether 
```suggestion
\cs_new:Npn \exp_args:NNNe #1#2#3#4 { \tex_expanded:D { \exp_not:n {#1#2#3} {#4} } }`
```

might be faster (but didn't benchmark)

Reply:
I thought about that myself - I was aiming for a def. that was similar to the other hand-tuned cases. (I do wonder about whether we should reduce those down: I suspect it would be better overall to avoid the potentially-riskign hand-defining even if things end up a tiny bit slower.)